package/yast2-rmt.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-rmt
-Version:        1.0.2
+Version:        1.0.3
```

Comment:
Maybe we should add a Dangerfile with a version check, same way as with RMT.

Reply:
Yes we should